clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/model/loadbalancer/EcsTargetGroup.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2017 Lookout, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.ecs.model.loadbalancer;
+
+import com.netflix.spinnaker.clouddriver.model.LoadBalancerProvider;
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class EcsTargetGroup implements LoadBalancerProvider.Details {
+  // TODO - add fields we want to persist from the cached value for target groups
```

Comment:
It seems to me this TODO is already done, isnt it?

Reply:
`EcsTargetGroup`/`EcsTargetGroupCacheClient` have not actually been used.

We may need to add fields to it later, when they are used.